packages/cli/internal/pkg/cli/account_activate.go

```diff
@@ -74,6 +75,7 @@ func (o *accountActivateOpts) Execute() error {
 	environmentVars := []string{
 		fmt.Sprintf("AGC_BUCKET_NAME=%s", o.bucketName),
 		fmt.Sprintf("CREATE_AGC_BUCKET=%t", !exists),
+		fmt.Sprintf("AGC_VERSION=%s", version.Version),
```

Comment:
How it will work when we upgrade AGC and re-activate the account?

Reply:
The resources will be retagged with the latest version